lua/lib/util.lua

```diff
@@ -46,6 +46,26 @@ util.file_exists = function(name)
   end
 end
 
+--- query file size.
+-- @tparam string name filepath
+-- @treturn number filesize in mb
+util.file_size = function(path)
+  if path ~= nil then
+    local f = io.open(path,"r")
+    if f~=nil then
+      local c = f:seek()      -- get current position
+      local s = f:seek("end") -- get file size
+      f:seek("set", c)        -- restore position
```

Comment:
it's not necessary to seek to the start, if you're closing the file, the first `seek()` call can be also safely omitted.